Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml

```diff
@@ -0,0 +1,1456 @@
+      version: -1
+      name: Verify Unusual Country
+      description: 'Verify that the ‘Suspicious_Elements’ playbook output contain ''Unusual Country'' '
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "324"
+      "yes":
+      - "256"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: Suspicious_Elements
+            iscontext: true
+          right:
+            value:
+              simple: Unusual Country
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3440,
+          "y": 770
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "324":
+    id: "324"
+    taskid: ea60761f-7aeb-4760-8129-2d395277efb9
+    type: regular
+    task:
+      id: ea60761f-7aeb-4760-8129-2d395277efb9
+      version: -1
+      name: Verify Playbook Output Error - Unusual Country
+      description: Prints an error entry with a given message
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      message:
+        simple: |-
+          The 'Suspicious_Elements' playbook output does not contain 'Unusual Country' value. One of the following may be the cause:
+          1- The 'Set' script failed to execute in the 'Set unusual country' task.
+          2- The 'key' input configuration was changed for the 'Set' automation used in the 'Set unusual country' task.
+          3- The 'value' input configuration was changed for the 'Set' automation used in the 'Set unusual country' task.
+          4- The 'append' input configuration was changed for the 'Set' automation used in the 'Set unusual country' task.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3440,
+          "y": 975
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "325":
+    id: "325"
+    taskid: 3eec7cbe-78e9-406e-87b4-8da1710b261c
+    type: condition
+    task:
+      id: 3eec7cbe-78e9-406e-87b4-8da1710b261c
+      version: -1
+      name: Verify Alert Search Results
+      description: Verify that the ‘alertsearchresults’ alert field was populated correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "326"
+      "yes":
+      - "305"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: alert.alertsearchresults
```

Comment:
Should it be `xdralertsearchresilts`?
![image](https://github.com/demisto/content/assets/88268646/54377cd6-5aa4-45c1-9e82-73f1f609dcba)
![image](https://github.com/demisto/content/assets/88268646/f35d33f9-c06d-4084-a86e-b23932feb954)